djangocms_versioning/plugin_rendering.py

```diff
@@ -24,7 +26,13 @@ def prefetch_versioned_related_objects(instance, toolbar):
             qs = versionable.content_model.objects.all()
         related_field = getattr(instance, field.name)
         if related_field:
-            filters = {versionable.grouper_field_name: related_field}
+            if isinstance(related_field, models.Manager):
+                filters = {}
+                filter_key = f'{versionable.grouper_field_name}__in'
```

Comment:
f-strings is not supported by python 3.5. 
```suggestion
                filter_key = '{{}}__in'.format(versionable.grouper_field_name)
```

Reply:
I will change that.